ledger/sandbox-classic/src/main/scala/platform/sandbox/Cli.scala

```diff
@@ -33,6 +33,12 @@ private[sandbox] object Cli extends SandboxCli {
           s" Also note that instructing $Name to load a scenario will have the side effect of loading _all_ the .dar files provided eagerly (see --eager-package-loading)."
       )
     parser
+      .opt[String]("sql-backend-jdbcurl")
```

Comment:
@SamirTalwar-DA Is there a good place to share this between the two sadbonxes but not with DAML on SQL?